aep/general/scope.yaml

```diff
@@ -4,13 +4,15 @@ order: 0
 categories:
   - code: meta
     title: Meta
+  - code: general
+    title: General
   - code: resources
     title: Resources
   - code: standard-methods
     title: Standard Methods
   - code: fields
     title: Fields
   - code: operations
-    title: Fields
```

Comment:
Should this, and the other unrelated changes (e.g. `"` -> `'`) be in a separate PR?  Not a _huge_ deal if you don't want to separate them out, but it's nice to keep the blame history clean.

Reply:
I'm okay with minor unrelated changes like these, but if you're adamant I can separate!

Reply:
Nah, I don't think this is the kind of "codebase" where it matters all that much.